### ISSUE TITLE
two types of "truncators"

### ISSUE THREAD

**Issue by MareoRaft** [ghseeli/trunconomics#2]
I feel that there are two types of truncators going on here and it may be helpful to distinguish them.  The first type of truncator I feel we have is truly a "truncator" in that it merely takes in a word and creates a truncation for it.  There second type is more like a "truncation finder" in that it takes in a word and a corpus, and then tries to find already extant truncations for the word within the corpus.

This isn't a concrete 'issue' but I wanted to bring it up.  We might benefit from choosing a naming scheme within our code to distinguish these.

**Comment by ghseeli** [comment]
I agree with this! The current truncator(s) are just meant to resemble some sort of baseline. I find when developing complex systems/algorithms, it can be easy to trick oneself into believing something is performing well when, in fact, it has no noticeable improvement over a common sense “baseline” solution. 

It might be worth breaking these into different folders for now? Something like you said: “truncators” versus “truncation-finders.” I think this will also help us clarify our thinking and development. I’ll think on this more, but if you have a clear idea in your head, I would be fine with you just carrying it out.

**Comment by MareoRaft** [comment]
(( As a side note: I have a similar issue with information input.  Sometimes we input a corpus (such as an article), and sometimes we merely input a vocabulary (a non-redundant list of words in no particular order).  I have distinguished these in fetch.py, but i got really confused trying to write things in main.py.  Sometimes the inputs we need differ (and i'm not talking about differing *format* or any semantics like that) depending on which truncator we're using.  sadness.  ))
